src/main/java/golf/servletquest/repository/StepRepository.java

```diff
@@ -0,0 +1,41 @@
+package golf.servletquest.repository;
+
+import golf.servletquest.entity.Step;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class StepRepository {
+    private List<Step> list;
```

Comment:
let's make it final and rename to `steps` as we should use the plural for list naming as it is an aggregation of items.